messaging-activemq/src/test/java/org/wildfly/extension/messaging/activemq/BridgeAddTestCase.java

```diff
@@ -0,0 +1,210 @@
+            model.get("static-connectors").add(CONNECTOR_NAME);
+
+            BridgeConfiguration bridgeConfiguration = BridgeAdd.createBridgeConfiguration(context, BRIDGE_NAME, model);
+
+            Assert.assertEquals(BRIDGE_NAME, bridgeConfiguration.getName());
+            Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfiguration.getQueueName());
+            Assert.assertEquals(CALL_TIMEOUT, bridgeConfiguration.getCallTimeout());
+            Assert.assertEquals(CONNECTOR_NAME, bridgeConfiguration.getStaticConnectors().get(0));
+        } finally {
+            System.clearProperty(BridgeAdd.CALL_TIMEOUT_PROPERTY);
+        }
+    }
+
+
+    /**
+     * Verifies that {@link BridgeAdd#createBridgeConfiguration(OperationContext, String, ModelNode)} creates
+     * a BridgeConfiguration instance with callTimeout attribute set to its' default value, when it's not modified via the management model
+     * attribute or the system property.
+     */
+    @Test
+    public void testCreateBridgeConfiguration_callTimeoutUnset() throws OperationFailedException {
+        ModelNode model = new ModelNode();
+        model.get("queue-name").set(SOURCE_QUEUE_NAME);
+        model.get("static-connectors").add(CONNECTOR_NAME);
+
+        BridgeConfiguration bridgeConfiguration = BridgeAdd.createBridgeConfiguration(context, BRIDGE_NAME, model);
+
+        Assert.assertEquals(BRIDGE_NAME, bridgeConfiguration.getName());
+        Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfiguration.getQueueName());
+        Assert.assertEquals(ActiveMQClient.DEFAULT_CALL_TIMEOUT, bridgeConfiguration.getCallTimeout());
+        Assert.assertEquals(CONNECTOR_NAME, bridgeConfiguration.getStaticConnectors().get(0));
+    }
+
+    @Test
+    public void testPerformRuntime() throws Exception {
+        ModelNode operation = Util.createAddOperation(BRIDGE_ADDRESS);
+
+        ModelNode model = new ModelNode();
+        model.get("call-timeout").set(CALL_TIMEOUT);
+        model.get("queue-name").set(SOURCE_QUEUE_NAME);
+        model.get("static-connectors").add(CONNECTOR_NAME);
+
+        BridgeAdd.INSTANCE.performRuntime(context, operation, model);
+
+        ArgumentCaptor<BridgeConfiguration> bridgeConfigurationCaptor = ArgumentCaptor.forClass(BridgeConfiguration.class);
+        Mockito.verify(activeMQServer).deployBridge(bridgeConfigurationCaptor.capture());
+        Assert.assertEquals(BRIDGE_NAME, bridgeConfigurationCaptor.getValue().getName());
+        Assert.assertEquals(CALL_TIMEOUT, bridgeConfigurationCaptor.getValue().getCallTimeout());
+        Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfigurationCaptor.getValue().getQueueName());
+        Assert.assertEquals(CONNECTOR_NAME, bridgeConfigurationCaptor.getValue().getStaticConnectors().get(0));
+    }
+
+    /**
+     * If the ActiveMQ server is not started, IllegalStateException is thrown.
+     */
+    @Test(expected = IllegalStateException.class)
```

Comment:
Doesn't have much value and is covered by the integration testing

Reply:
Files removed.